### ISSUE TITLE
wofi vs rofi

### ISSUE THREAD

**Issue by mindrunner** [fdw/rofi-rbw#66]
Using `rofi`, I can chose via keybinds what to copy (user, pass, otp, type, etc...). Is there an equivalent for `wofi`?

I think `wofi` lacks this feature sort of, right? I am currently using a combination of the official `bitwarden` `cli` and bash script for this. after selecting an entry, I can select what to copy. Would nice to see something like that in `rofi-rbw` as well. Or am I missing something?

**Comment by fdw** [comment]
> I think wofi lacks this feature sort of, right?

Yup, `wofi` doesn't do custom shortcuts, as far as I know, sorry.

> I am currently using a combination of the official bitwarden cli and bash script for this. after selecting an entry, I can select what to copy. Would nice to see something like that in rofi-rbw as well. Or am I missing something?

If I understand you right, you want to select an entry and then be presented with a menu which "part" (f.e. username, password, url...) you want to copy/type? Does `--targets menu` work for you?
Also, if you can use `rofi`, you might be interested in the branch `custom-shortcuts` 🙂

**Comment by mindrunner** [comment]
> Does --targets menu work for you?

Shame on me. That's exactly what I am looking for... However. It seems buggy, in some cases (Secure note with custom fields. The `menu` is an empty list)

> Also, if you can use rofi, you might be interested in the branch

I use `wofi` for so many things and prefer to have everything themed the same way. Also `rofi` is not really wayland...

**Comment by fdw** [comment]
> However. It seems buggy, in some cases (Secure note with custom fields. The menu is an empty list)

I can reproduce it, but unfortunately, it's a bug (or missing feature) in [rbw itself](https://github.com/doy/rbw/). Using the cli, I can't access the fields of a note at all. You can open a bug over there and I'll see that I'll add it to `rofi-rbw` when it works.

> I use wofi for so many things and prefer to have everything themed the same way. Also rofi is not really wayland...

I see 🙂 I wasn't sure if `rofi` is an option for you, that's why I suggested it.

**Comment by mindrunner** [comment]
https://github.com/doy/rbw/issues/101

**Comment by fdw** [comment]
Thank you! I've subscribed to the issue, so I'd close this ticket, if it's alright for you 🙂
